packages/auth-password/index.js

```diff
@@ -12,6 +12,9 @@ class PasswordAuthStrategy {
       identityField: 'email',
       secretField: 'password',
       protectIdentities: false,
+      identityFilter: ({ identityField, identity }) => ({
+        [identityField]: identity,
```

Comment:
This would be very easy to accidentally overwrite and completely break sign in.

Also, what about `auth-passport`? Does it need this kind of thing too?

Reply:
if we have login hook, (before issuing token/cookies) all auth strategy would be covered not just password. then admin-ui may have another hook for preventing users from login to actual admin-ui pages not just login. 

in ks4, if you are not allowed to login to admin-ui, it would just sit on login page saying, you are already logged in, do you want to sign out (something like that), similar solution would be much appreciated. I earlier suggested to have `redirectUrl` for admin-ui options in case user does not have admin-ui access or they are trying to access admin-ui only for authentication.